src/components/FilterPanel.jsx

```diff
@@ -73,9 +93,43 @@ export default function FilterPanel({ onClose }) {
             <Typography>{name}</Typography>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
       ))}
+
+      {filtersChanged && (
```

Comment:
I'm noticing the buttons shift when this appears/disappears. How about we keep this element but just change the text and/or icon when filters are currently applied? You could use the **InfoOutlined** or **CheckCircleOutLine** icon to match with the warning one, show  and display something like "Your filters are currently applied".